autogen/agenteval/agent_eval.py

```diff
@@ -0,0 +1,101 @@
+import autogen
+from criterion import Criterion
+from critic_agent import CriticAgent
+from quantifier_agent import QuantifierAgent
+from subcritic_agent import SubCriticAgent
+from task import Task
+
+import os
+import sys
+from typing import Callable, Dict, Optional, Union
+
+
+def generate_criteria(
```

Comment:
this function could technically be part of critic_agent - with some caveats.

Reply:
if the class itself doesnt provide any specialization then why not just do `critic_agent = ConversableAgent(system_prompt=[insert special prompt here])` but this might be more of a philosophical autogen question...

Reply:
Right now it doesn't have any special code, but there were talks about adding additional functionality (user defined functions, more configuration, etc) that I did not implement in this initial change. It also allows for users to use the critic/quantifier agents standalone without needing to come up with their own prompt.